docs/tox.ini

```diff
@@ -47,6 +49,40 @@ commands =
   make --directory=sphinx clean
   make --directory=sphinx json SPHINXOPTS="-W --keep-going"
 
+[testenv:sphinx-mdx]
```

Comment:
copy of `sphinx` env but specific to mdx so that we don't conflict with legacy json build.